shared/settings/index.tsx

```diff
@@ -44,6 +45,7 @@ const mergeProps = (stateProps: ReturnType<typeof mapStateToProps>, dispatchProp
   badgeNotifications: stateProps.badgeNotifications,
   badgeNumbers: stateProps._badgeNumbers.toObject(),
   children: ownProps.children,
+  contactsLabel: stateProps._contactImportEnabled ? 'Contacts' : 'Import contacts',
```

Comment:
Can you capitalize the C in `Import contacts` to match the other things in this menu?